Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -47,38 +40,67 @@ class LibraryViewController: UIViewController {
         return librarySegmentControl
     }()
 
-    fileprivate lazy var titleLabel: UILabel = {
-        let titleLabel = UILabel()
-        titleLabel.font = DynamicFontHelper.defaultHelper.DefaultStandardFontBold
-        titleLabel.textColor = UIColor.theme.tabTray.tabTitleText
-        titleLabel.textAlignment = .center
-        titleLabel.numberOfLines = 1
-        return titleLabel
+    lazy var navigationToolbar: UIToolbar = {
+        let toolbar = UIToolbar()
+        toolbar.delegate = self
+        toolbar.setItems([UIBarButtonItem(customView: librarySegmentControl)], animated: false)
+
+        return toolbar
     }()
 
-    fileprivate lazy var topLeftButton: UIButton =  {
-        let button = UIButton()
-        button.setTitle(Strings.BackTitle, for: .normal)
-        button.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .regular)
-        button.setTitleColor(UIColor.systemBlue, for: .normal)
-        button.titleLabel?.textAlignment = .left
-        button.addTarget(self, action: #selector(topLeftButtonAction), for: .touchUpInside)
+    fileprivate lazy var topLeftButton: UIBarButtonItem =  {
+        let button = UIBarButtonItem(image: UIImage.templateImageNamed("goBack"),
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(topLeftButtonAction))
+        button.accessibilityIdentifier = "libraryPanelTopLeftButton"
         return button
     }()
-    
-    fileprivate lazy var topRightButton: UIButton =  {
-        let button = UIButton()
-        button.setTitle(String.AppSettingsDone, for: .normal)
-        button.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .regular)
-        button.setTitleColor(UIColor.systemBlue, for: .normal)
-        button.addTarget(self, action: #selector(topRightButtonAction), for: .touchUpInside)
+
+    fileprivate lazy var topRightButton: UIBarButtonItem =  {
+        let button = UIBarButtonItem(title: String.AppSettingsDone,
+                                     style: .done,
+                                     target: self,
+                                     action: #selector(topRightButtonAction))
+        button.accessibilityIdentifier = "libraryPanelTopRightButton"
         return button
```

Comment:
My suggestion would be to avoid doing this type of styling  mainly because XCode will auto wrap and this makes searching easier too

```suggestion
    fileprivate lazy var topRightButton: UIBarButtonItem =  {
        let button = UIBarButtonItem(title: String.AppSettingsDone, style: .done, target: self, action: #selector(topRightButtonAction))
        button.accessibilityIdentifier = "libraryPanelTopRightButton"
        return button
```